addons/l10n_at/data/template/account.account-at.csv

```diff
@@ -1,198 +1,235 @@
+"chart_at_template_0830","Other investments","0830","False","asset_non_current","l10n_at.account_tag_external_code_0830","Sonstige Beteiligungen"
+"chart_at_template_0840","Loans to affiliated companies","0840","False","asset_non_current","l10n_at.account_tag_external_code_0840","Ausleihungen an verbundene Unternehmen"
+"chart_at_template_0850","Loans to other long-term investees and investors","0850","False","asset_non_current","l10n_at.account_tag_external_code_0850","Ausleihungen an Unternehmen, mit denen ein Beteiligungsverhältnis besteht"
+"chart_at_template_0860","Other Loans","0860","False","asset_non_current","l10n_at.account_tag_external_code_0860","Sonstige Ausleihungen"
+"chart_at_template_0870","Shares in corporations of a non-participating nature","0870","False","asset_non_current","l10n_at.account_tag_external_code_0870","Anteile an Kapitalgesellschaften ohne Beteiligungscharakter"
+"chart_at_template_0880","Shares in partnerships of a non-participating nature","0880","False","asset_non_current","l10n_at.account_tag_external_code_0880","Anteile an Personengesellschaften ohne Beteiligungscharakter"
+"chart_at_template_0900","Cooperative shares of a non-participating nature","0900","False","asset_non_current","l10n_at.account_tag_external_code_0900","Genossenschaftsanteile ohne Beteiligungscharakter"
+"chart_at_template_0910","Shares in investment funds","0910","False","asset_non_current","l10n_at.account_tag_external_code_0910","Anteile an Investmentfonds"
+"chart_at_template_0980","Prepayments for financial assets","0980","False","asset_prepayments","l10n_at.account_tag_external_code_0980","Geleistete Anzahlungen für Finanzanlagen"
+"chart_at_template_0990","Accumulated Depreciation","0990","False","asset_non_current","l10n_at.account_tag_external_code_0990","Kumulierte Abschreibungen"
+"chart_at_template_1600","Merchandise","1600","False","asset_current","l10n_at.account_tag_external_code_1600","Handelswaren"
+"chart_at_template_1800","Prepayments for inventories 20%","1800","False","asset_prepayments","l10n_at.account_tag_external_code_1800","Geleistete Anzahlungen auf Vorräte 20 %"
+"chart_at_template_1801","Prepayments for inventories 10%","1801","False","asset_prepayments","l10n_at.account_tag_external_code_1801","Geleistete Anzahlungen auf Vorräte 10 %"
+"chart_at_template_1803","Prepayments for inventories 0%","1803","False","asset_prepayments","l10n_at.account_tag_external_code_1803","Geleistete Anzahlungen auf Vorräte 0 %"
+"chart_at_template_2000","Trade receivables, domestic","2000","True","asset_receivable","l10n_at.account_tag_external_code_2000","Forderungen aus Lieferungen und Leistungen Inland"
+"chart_at_template_2099","Trade receivables, domestic (point of sale)","2099","True","asset_receivable","l10n_at.account_tag_external_code_2000","Forderungen aus Lieferungen und Leistungen Inland (Point Of Sale)"
```

Comment:
Point of sale not translated ?

Reply:
It was not translated previously (all German version). I think the term is also used in Germany.